### PR TITLE
Getting Started to use Chat API not Completions 

### PR DESCRIPTION
**Pull request by lunamidori5**
**Description**
Getting Started Pages using outdated API, brings them upto OpenAI V1 Chat API

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by lunamidori5** [comment]
@mudler Can you check this out <3

File: docs/content/getting_started/_index.en.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ docker run -p 8080:8080 -v $PWD/models:/models -ti --rm quay.io/go-skynet/local-
 # └───────────────────────────────────────────────────┘
 
 # Try the endpoint with curl
-curl http://localhost:8080/v1/completions -H "Content-Type: application/json" -d '{
+curl http://localhost:8080/v1/chat/completions -H "Content-Type: application/json" -d '{
      "model": "your-model.gguf",
      "prompt": "A long time ago in a galaxy far, far away",
```
Comment (mudler) [inline docs/content/getting_started/_index.en.md:47]: this doesn't look right, the chat completion doesn't actually use a prompt, but it expects a list of messages

Comment (lunamidori5) [reply docs/content/getting_started/_index.en.md:47]: @mudler Your right Ill fix it thank you!

**Comment by lunamidori5** [comment]
@mudler fixed! Thank you for pointing that out!

**Comment by lunamidori5** [comment]
@mudler please and thank you... <3

**Pull request closed without merging**